alf/experience_replayers/replay_buffer.py

```diff
@@ -291,6 +329,13 @@ def add_batch(self, batch, env_ids=None, blocking=False):
                 self._indexed_pos[(env_ids[epi_first],
                                    self.circular(overwriting_pos[epi_first])
                                    )] = overwriting_pos[epi_first]
+                epi_last, = torch.where(step_types == ds.StepType.LAST)
+                if self._keep_episodic_info and self._record_episodic_return:
+                    # Backfill episodic returns for episodes which ended
+                    self._compute_store_episodic_return(
```

Comment:
no need to do this if epi_last is empty.

Reply:
done